tools/kmod/sof_remove.sh

```diff
@@ -30,6 +38,11 @@ remove_module snd_sof_pci_intel_tgl
 remove_module snd_sof_acpi_intel_byt
 remove_module snd_sof_acpi_intel_bdw
 
+#-------------------------------------------
+# temporary dependency
+#-------------------------------------------
+try_remove_module snd_sof_intel_hda_common
```

Comment:
If this is temporary then the new function is overkill, this is enough and IMHO faster to understand because no indirection:
```shell
( set +e; remove_module snd_sof_intel_hda_common )
```